packages/@ionic/v1-toolkit/src/lib/gulp.ts

```diff
@@ -1,15 +1,49 @@
+'use strict';
+
 import { prettyPath } from '@ionic/cli-framework/utils/format';
```

Comment:
Shouldn't be necessary as TypeScript writes this to the emitted JS.

```suggestion
import { prettyPath } from '@ionic/cli-framework/utils/format';
```